### DISCUSSION TITLE
blur filter unique to the scean

### DISCUSSION THREAD

**Discussion by BJSam** [Xaymar/obs-stream-effects#963] [Ideas & Feedback]
Hi, I was trying to blur a source which is a scene but the blur effect applied to the main scene, I wanted it to be applied to that particular scene where I have added the filter;
example:
I have a scene called mainMeet and a scene called displayTxt. so, this displayTxt scene has a scene source of mainMeet and if I add a blur filter to this scene (source scene in displayTxt which has mainMeet ) then the original scene and the imported scene is getting blurred. can we make only one scene (displayTxt) is blurred not the mainMeet i.e when I go to mainMeet blur should not be there and when I move back to displayTxt  then blur should appear.

![image](https://user-images.githubusercontent.com/20676057/197398031-663be85d-8e6a-4155-8bc1-aabf8389a933.png)

**Comment by Xaymar** [comment]
Duplicate the Scene or use Source Mirror. We can't affect how OBS Studio works, just extend it within its own limits.

**Reply by BJSam** [reply]
> Duplicate the Scene or use Source Mirror. We can't affect how OBS Studio works, just extend it within its own limits.

so can we add a condition saying if source and scene then add blur. like identifying where to blur based on the source and scene; instead of only the source.

**Reply by Xaymar** [reply]
No, duplicate the Source/Scene using Source Mirror, then use Blur on the Source Mirror. Without rewriting how OBS Studio works entirely, this is impossible to do.
